game_eggs/cod2/egg-call-of-duty2.json

```diff
@@ -0,0 +1,102 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-04-05T12:31:07+02:00",
+    "name": "Call of Duty 2",
+    "author": "sebastijantkalcec@gmail.com",
+    "description": null,
+    "features": null,
+    "docker_images": {
+        "bacacro\/cod2pterodactyl:latest": "bacacro\/cod2pterodactyl:latest"
+    },
+    "file_denylist": [],
+    "startup": "export LD_LIBRARY_PATH=\/home\/container; export LD_PRELOAD=\"libcod.so\"; .\/cod2_lnxded +set sv_cracked {{IS_CRACKED}} +set sv_pure {{IS_PURE}} +set sv_punkbuster {{PUNKBUSTER}} +set dedicated {{IS_DEDICATED}} +set net_ip 0.0.0.0 +set net_port {{SERVER_PORT}} +set sv_maxclients {{MAX_PLAYERS}} +set g_gametype {{GAMETYPE}} +map {{DEFAULT_MAP}} +exec server.cfg;",
```

Comment:
+set sv_cracked {{IS_CRACKED}} ??? Thats not allowed here

Reply:
Sorry my bad.

Reply:
Sorry, my bad